content/artifacts/artifacts-finder/plugin.js

```diff
@@ -200,3 +230,9 @@ class ArtifactsFinder {
 }
 
 export default ArtifactsFinder;
+
+function distance(from, to) {
+  let fromloc = from.location;
+  let toloc = to.location;
+  return Math.sqrt((fromloc.coords.x - toloc.coords.x) ** 2 + (fromloc.coords.y - toloc.coords.y) ** 2);
+}
```

Comment:
Perhaps this code block code be move before the ArtifactsFinder class with the rest of the utility functions at the top of the file, to maintain the readability in the file.